src/features/usage-consolidation-accordion/usage-consolidation-content-title.js

```diff
@@ -163,7 +164,12 @@ const UsageConsolidationContentTitle = ({
     formatter,
   };
 
-  return (
+  return noCostPerUseAvailable ? (
```

Comment:
Do we also need to check here that platforms are not empty?

Reply:
Don't know, haven't seen description of the same behavior, but thanks, I'll add it, to omit same bug in future